examples/simple/src/main.rs

```diff
@@ -109,7 +108,6 @@ impl<'s> ApplicationHandler for SimpleVelloApp<'s> {
             WindowEvent::Resized(size) => {
                 self.context
                     .resize_surface(&mut render_state.surface, size.width, size.height);
-                render_state.window.request_redraw();
```

Comment:
:+1: Requesting redraw when resized is implicit, right?

Reply:
Should be, if not, it's a bug